GettingStartedWithTheMRTK の v2.1.0対応 (#48) (#65)
HoloLabInc/MixedRealityToolkit-Unity: Documentation.ja/GettingStartedWithTheMRTK.md

  MRTKは、 Unity 2018 で IL2CPP と .NET scripting backends の両方をサポートします。

* [最新の MRTK release](https://github.com/Microsoft/MixedRealityToolkit-Unity/releases)
* [Windows SDK 18362+](https://developer.microsoft.com/en-US/windows/downloads/windows-10-sdk)

  これは WMR、HoloLens 1、または HoloLens 2 向けの UWP アプリを構築する場合に必要です。OpenVR 向けに構築する場合は不要です。

## チュートリアルを始める

> MRTK、または MR 開発が初めての場合は、MRTKv2 を使った [チュートリアルを始める](https://docs.microsoft.com/en-us/windows/mixed-reality/mrlearning-base)をチェックすることをお勧めします。コアとなるビルディングブロックについては、[MRTK 101: How to use Mixed Reality Toolkit Unity for Basic Interactions (HoloLens 2, HoloLens, Windows Mixed Reality, Open VR)](https://docs.microsoft.com/ja-jp/windows/mixed-reality/mrtk-101) をチェックしてください。

Reviewer: ```suggestion
MRTK、または MR 開発が初めての場合は、MRTK v2 を使った [チュートリアルを始める](https://docs.microsoft.com/en-us/windows/mixed-reality/mrlearning-base)をチェックすることをお勧めします。コアとなるビルディング ブロックについては、[MRTK 101: How to use Mixed Reality Toolkit Unity for Basic Interactions (HoloLens 2, HoloLens, Windows Mixed Reality, Open VR)](https://docs.microsoft.com/ja-jp/windows/mixed-reality/mrtk-101) をチェックしてください。
```

## MRTK を Unity Project に追加する

> Mixed Reality Toolkitは、Nuget.org からダウンロードできます。詳細は [MRTK NuGet Package](MRTKNuGetPackage.md) を参照してください。

Reviewer: ```suggestion
Mixed Reality Toolkit は、Nuget.org からダウンロードできます。詳細は [MRTK NuGet Package](MRTKNuGetPackage.md) を参照してください。
```

シーン内を移動する：

* **W/A/S/D** キーを使用して、カメラを前後/左右に移動します。

> * **Q/E** to move the camera veritically.

Reviewer: ```suggestion
* **Q/E** キーを使用して、カメラを上下に動かします。
```

* **マウスの右ボタン**を押したままにして、カメラを回転させます。

手の入力をシミュレートする：

* **Space キー**を押し続けて、右手を有効にします。
* Space キーを押しながら、マウスを動かして手を動かします。
* **マウスのスクロールホイール**を使用して、手の奥行を調整します。

> * **マウスの左ボタン**をクリックして、ジェスチャを切り替えます。

Reviewer: ```suggestion
* **マウスの左ボタン**をクリックして、ピンチ ジェスチャをシミュレートします。
```

お勧めの次のステップを紹介します。

> * [MRTK 101: How to use Mixed Reality Toolkit Unity for Basic Interactions](https://docs.microsoft.com/ja-jp/windows/mixed-reality/mrtk-101) では、グラブ、移動、スケール、回転などの一般的な空間的インタラクションの実現方法について学ぶ。

Reviewer: ```suggestion
* [MRTK 101: How to use Mixed Reality Toolkit Unity for Basic Interactions](https://docs.microsoft.com/ja-jp/windows/mixed-reality/mrtk-101) で、グラブ、移動、スケール、回転などの一般的な空間的インタラクションの実現方法について学ぶ。
```

* [UI とインタラクションのビルディング ブロック](../README.md#ui-and-interaction-building-blocks) で UX コントロール について学ぶ。

> * [MRTK Examples Hub](README_ExampleHub.md) (事前にビルドされたアプリケーションパッケージはリリースページに含まれています) を試す。

Reviewer: ```suggestion
* [MRTK Examples Hub](README_ExampleHub.md) を試す。(事前にビルドされたアプリケーション パッケージはリリース ページに含まれています)
```

> * [MRTK Examples Hub](README_ExampleHub.md) を試す。(事前にビルドされたアプリケーション パッケージはリリース ページに含まれています)

Reviewer: ```suggestion
* [MRTK Examples Hub](README_ExampleHub.md) を試す(事前にビルドされたアプリケーション パッケージはリリース ページに含まれています)。
```

* [Mixed Reality Toolkit プロファイル設定ガイド](MixedRealityConfigurationGuide.md) で MRTK Configuration profile の使い方を学ぶ。

> * [アーキテクチャの全体像](./Architecture/Overview.md) を、学ぶ。

Reviewer: ```suggestion
* [MRTK のアーキテクチャ](./Architecture/Overview.md) を、学ぶ。
```

> * [入力の概要](./Input/Overview.md) を、学ぶ。

Reviewer: ```suggestion
* [MRTK のインプット システム](./Input/Overview.md) を、学ぶ。
```

> * 複合現実感の設計と開発を強化する [MRTK's Tools](../README.md#tools) について学ぶ。

Reviewer: ```suggestion
* 複合現実感の設計と開発を強化する [MRTK のツール](../README.md#tools) について学ぶ。
```

> * [入力シミュレーションサービス (Input Simulation Service)](InputSimulation/InputSimulationService.md) を読み、エディタでハンド入力をシミュレートする方法を学ぶ。

Reviewer: ```suggestion
* [インプット シミュレーション ガイド](InputSimulation/InputSimulationService.md) を読み、エディターでハンド入力をシミュレートする方法を学ぶ。
```
